modules/postgres/postgres.go

```diff
@@ -18,6 +19,10 @@ const (
 	defaultSnapshotName  = "migrated_template"
 )
 
+// SQLDriverName is passed to sql.Open() to connect to the database when making or restoring snapshots.
+// This can be set if your app imports a different postgres driver, f.ex. "pgx"
+var SQLDriverName = "postgres"
```

Comment:
I agree, and I had this comment pending in my list. Thanks for raising your concerns around it.

Yes, setting this as global would mean users need to set and reset the value per test, making parallel testing flaky because of potential race conditions. I'd go with what @ash2k suggested, defining the driver name as an attribute of each container. So creating a functional option that sets the driver name seems reasonable to me: e.g. `postgres.WithDriver("pgx")`. The docs for that function should relate the default driver value if the option is not used.